ci/scripts/model_health_check.py

```
@@ -0,0 +1,219 @@
+                for f in sorted(set(files)):
+                    print(f"    - {f}")
+        else:
+            label = f"HTTP {status}" if status > 0 else "ERROR"
+            print(f"  FAIL  {model} -> {label}: {detail}")
+            failures.append((model, status, detail, files))
+
+    print()
+
+    if failures:
+        print(f"{len(failures)} model(s) unreachable:\n")
+        for model, status, detail, files in failures:
```

Ruff (0.15.2)

[warning] Loop control variable `detail` not used within loop body

(B007)